pac4j-oauth/src/main/java/org/pac4j/oauth/profile/google2/Google2ProfileDefinition.java

```diff
@@ -19,23 +19,23 @@
  */
 public class Google2ProfileDefinition extends OAuth20ProfileDefinition<Google2Profile, OAuth20Configuration> {
 
-    public static final String DISPLAY_NAME = "displayName";
-    public static final String GIVEN_NAME = "name.givenName";
-    public static final String FAMILY_NAME = "name.familyName";
-    public static final String URL = "url";
-    public static final String PICTURE = "image.url";
-    public static final String LANGUAGE = "language";
-    public static final String EMAIL = "email";
+    public static final String EMAIL_VERIFIED = "email_verified";
+    public static final String GIVEN_NAME = "given_name";
+    public static final String NAME = "name";
+    public static final String PICTURE = "picture";
+    public static final String PROFILE = "profile";
 
     public Google2ProfileDefinition() {
         super(x -> new Google2Profile());
-        primary(DISPLAY_NAME, Converters.STRING);
+        primary(EMAIL, Converters.STRING);
+        primary(EMAIL_VERIFIED, Converters.BOOLEAN);
+        primary(CommonProfileDefinition.FAMILY_NAME, Converters.STRING);
+        primary(GENDER, Converters.GENDER);
         primary(GIVEN_NAME, Converters.STRING);
-        primary(FAMILY_NAME, Converters.STRING);
-        primary(URL, Converters.URL);
+        primary(LOCALE, Converters.LOCALE);
```

Comment:
The EMAIL,FAMILY_NAME,GENDER,LOCALE is already defined in super (CommonProfileDefinition)